maps/sccv_horizon/sccv_horizon-2_deck_2.dmm

```diff
@@ -44806,7 +44806,7 @@
 	input_tag = "cooling_in";
 	name = "Engine Cooling Control";
 	output_tag = "cooling_out";
-	sensors = list("engine_sensor"="Engine Core")
+	sensors = list("engine_sensor"="Engine  Core")
```

Comment:
why?

Reply:
It's some error with SDMM I think. It happens every time I open deck two. Thought I caught this.

Reply:
Fixed now.